components/job-orchestration/job_orchestration/executor/query/utils.py

```diff
@@ -0,0 +1,135 @@
+import datetime
+import os
+import signal
+import subprocess
+import sys
+from contextlib import closing
+from logging import Logger
+from pathlib import Path
+from typing import Any, Dict, List
+
+from clp_py_utils.clp_config import QUERY_TASKS_TABLE_NAME
+from clp_py_utils.sql_adapter import SQL_Adapter
+from job_orchestration.scheduler.scheduler_data import QueryTaskResult, QueryTaskStatus
+
+
+def get_task_log_file_path(clp_logs_dir: Path, job_id: str, task_id: int) -> Path:
+    worker_logs_dir = clp_logs_dir / job_id
+    worker_logs_dir.mkdir(exist_ok=True, parents=True)
+    return worker_logs_dir / f"{task_id}-clo.log"
+
+
+def report_command_creation_failure(
```

Comment:
making this to be an individual function might be an overkill
@kirkrodrigues any suggestions?

Reply:
It's fine